index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    <script>
+    function myFunction() {
+        innerhtml = "Paragraph changedm, again";
+        x = 7
+        y = 8
+        z = x*y
+        document.getElementById("demo").innerHTML = z;
+        }
+    </script>
+    <p id="demo">A Paragraph</p>
+    <button type="button" onclick="myFunction()">Try it</button>
+
+    <!-- Insert this script at the bottom of the HTML, but before you use any Firebase services -->
+    <!-- Add the entire Firebase JavaScript SDK -->
+    <script src="https://www.gstatic.com/firebasejs/8.8.1/firebase.js"></script>
+    <script>
+    // Your web app's Firebase configuration
+    // For Firebase JS SDK v7.20.0 and later, measurementId is optional
+    var firebaseConfig = {
+        apiKey: "AIzaSyAywIdDnnW2oSosheG6aYhdMVmGGpa5zHQ",
+        authDomain: "ascola-nike-spartner.firebaseapp.com",
+        databaseURL: "https://ascola-nike-spartner-default-rtdb.firebaseio.com",
+        projectId: "ascola-nike-spartner",
+        storageBucket: "ascola-nike-spartner.appspot.com",
+        messagingSenderId: "760534753646",
+        appId: "1:760534753646:web:995efaa259584c9ea1617f",
+        measurementId: "G-BS3664FSCB"
+    };
+    // Initialize Firebase
+    firebase.initializeApp(firebaseConfig);
+    firebase.analytics();
+    </script>
+
+    <script>
+    var user01ref = firebase.database().ref("Users/user01");
+    var user01URL = user01ref.toString();
+    function FBFunction() {

```

Comment:
lets see if the function can return the VALUE of the URL. means the output should be : 
`{
  "name" : "beer.twito",
  "sport" : [ "running", "biking" ]
}
`